youtube/templates/base.html

```diff
@@ -3,7 +3,9 @@
     <head>
         <meta charset="utf-8">
         <title>{{ page_title }}</title>
-        <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; media-src 'self' https://*.googlevideo.com">
+        {% if settings.proxy_images %}
+            <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; media-src 'self' https://*.googlevideo.com">
```

Comment:
Don't remove the CSP completely, it's there to make sure bugs/exploits don't cause the browser to make non-anonymous requests.